src/functions-and-arrays.js

```diff
@@ -1,19 +1,95 @@
 // Iteration #1: Find the maximum
 
+
+maxOfTwoNumbers = (number1, number2) => {
+  if(number1 > number2)
+    return number1;
+  else {
+    return number2;
+  }
+}
+
+maxOfTwoNumbers(4, 3);
+
+// Second way of doing it
+// function maxOfTwoNumbers(number1, number2) {
+//   const result = Math.max(number1 , number2);
+//   return result
+// }
+
+// maxOfTwoNumbers(3, 4);
+
+
+
 // Iteration #2: Find longest word
 const words = ['mystery', 'brother', 'aviator', 'crocodile', 'pearl', 'orchard', 'crackpot'];
 
-// Iteration #3: Calculate the sum
+let longestWord;
```

Comment:
`let longestWord` should be defined inside function findLongestWord as it only needs to be used within the scope of that function